tests/playwright/pages/wp-admin-page.js

```diff
@@ -20,20 +18,60 @@ exports.WpAdminPage = class wpAdminPage {
 		}
 
 		await this.page.waitForSelector( 'text=Log In' );
-		await this.page.fill( 'input[name="log"]', 'admin' );
-		await this.page.waitForTimeout( 500 );
-		await this.page.fill( 'input[name="pwd"]', 'password' );
+		await this.page.fill( 'input[name="log"]', this.config.user.username );
+		await this.page.fill( 'input[name="pwd"]', this.config.user.password );
```

Comment:
Why it's here? There is already a login in the global setup

Reply:
I remove any login, from the test, but let leave the method for now.